lib/src/native/realm_core.dart

```diff
@@ -743,6 +751,14 @@ extension _StringEx on String {
     nativeString.last = 0; // zero terminate
     return result.cast();
   }
+
+  Pointer<realm_string_t> toRealmString(Allocator allocator) {
```

Comment:
I feel this belongs somewhere else.. not really relates to `LastError`

Reply:
It is in `extension _StringEx on String {`